hdat/main.py

```diff
@@ -109,9 +109,9 @@ def show_result(suites, result):
     suite = select_suite(suites, result['suite_id'])
     try:
         suite.show(result)
-    except Exception:
+    except Exception as e:
         traceback.print_exc()
-        msg = 'Error when attempting to show "{}"'
```

Comment:
It seems a little odd to use string concatenation and string formatting.  Perhaps:

```
msg = 'Error when attempting to show "{}": {}'
raise AbortError(msg.format(print_resultspec(result), e))
```

would be better?